cosmos-tidoop-api/README.md

```diff
@@ -0,0 +1,209 @@
+    $ npm install
+    
+That must download all the dependencies under a `node_modules` directory.
+
+[Top](#top)
+
+###<a name="section2.3."></a>MySQL database installation
+Use the file `resources/mysql_db_and_tables.sql` for creating both `cosmos` database (if not yet existing) and `tidoop_job` table.
+
+    $ mysql -u <mysql_user> -p < resources/mysql_db_and_tables.sql
+
+The `tidoop_job` table tracks, for each MapReduce job:
+
+* The job id, with format `tidoop_job_<timestamp>`.
+* The class name of the MapReduce job to be run.
+* The timestamp when the job was launched.
+* The timestamp when the job finished.
+* The mapping progress, in percentage.
+* The reducing progress, in percentage.
+
+[Top](#top)
+
+###<a name="section2.4"></a>Unit tests
+To be done.
+
+[Top](#top)
+
+##<a name="section3"></a>Configuration
+cosmos-tidoop-api is configured through a JSON file (`conf/cosmos-tidoop-api.json`). These are the available parameters:
+
+* **host**: FQDN or IP address of the host running the service. Do not use `localhost` unless you want only local clients may access the service.
+* **port**: TCP listening port for incomming API methods invocation. 12000 by default.
+* **mysql**:
+    * **host**: FQDN or IP address of the host running the service.
+    * **port**: TCP listening port for the MySQL service, typically 3306.
+    * **user**: A valid user allowed to write and read the MySQL database.
+    * **password**: Password for above user.
+    * **database**: Database used to track information regarding the launched MR jobs; "cosmos" by default.
+* **log**:
+    * **file_name**: path of the file where the log traces will be saved in a daily rotation basis. This file must be within the logging folder owned by the the user `tidoop`.
+    * **date_pattern**: data pattern to be appended to the log file name when the log file is rotated.
+
+[Top](#top)
+
+##<a name="section4"></a>Running
+The Http server implemented by cosmos-tidoop-api is run as (assuming your current directory is `fiware-cosmos/cosmos-tidoop-api`):
+
+    $ npm start
+    
+If everything goes well, you should be able to remotely ask (using a web browser or `curl` tool) for the version of the software:
+
+    $ curl -X GET "http://<host_running_the_api>:12000/tidoop/v1/version"
+    {"version": "0.1.1"}
+    
+cosmos-tidoop-api typically listens in the TCP/12000 port, but you can change if by editing `conf/cosmos-tidoop-api.conf` as seen above.
+
+[Top](#top)
+
+##<a name="section5"></a>Usage
+Please refer to this [Apiary](http://docs.tidoopmrlibapi.apiary.io/#) documentation.
+
+Please observe the MapReduce jobs usually take some time to return a result. This is why the cosmos-tidoop-api operations run the job, but do not return any result, except a `200 OK` (if the job could be successfully run) and a job identifier. In order to get the result of the opearion (or its progress), such a job identifier must be used for querying the API later.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/a46e1731daedd1c5d9c75e1ca08e90be37aff51a